lektor/admin/modules/serve.py

```diff
@@ -1,7 +1,8 @@
+from __future__ import absolute_import
 import os
 import mimetypes
 import posixpath
-from cStringIO import StringIO
```

Comment:
cStringIO is a byte IO on python 2.  The correct type for Python 3 is BytesIO.  Changing this to a Python 3 StringIO means that the code completely changes behavior which is wrong.